packages/serp-report/src/content_scripts/serp-report.js

```diff
@@ -184,7 +184,10 @@ function removeWheel(anchor) {
         return;
       }
 
-      if (message.data.startsWith('WTMReportResize')) {
+      if (
```

Comment:
```suggestion
      if (String(message.data).startsWith('WTMReportResize'))
```

Reply:
Is it better? Given that the input can be arbitrary data, I find the explicit check for `string` easier to reason about. For instance, `['WTMReportResize']` would be detected by the `typeof` check but not by the `String` constructor.

Reply:
The type check is more correct, but I think it is not necessary here. As I wrote my suggestion is only cosmetic. The code is shorter.